pkg/certificate/providers/config.go

```diff
@@ -66,8 +66,42 @@ func NewCertificateManager(ctx context.Context, kubeClient kubernetes.Interface,
 				Provider:    option.AsProviderSpec(),
 				TrustDomain: trustDomain,
 			},
+			Intent: constants.MRCIntentPassive,
```

Comment:
nit: I think the intent here could be active, but I don't think this value impacts the functionality of the mrcCompatClient.